README.md

```diff
@@ -1 +1,108 @@
+
+### Step 4
+To ensure the Dev Container is running properly, look back at the bottom left corner of the VS code where the icon was. It should look like two arrowheads sliding past each other, as seen in the picture below.
+
+<img src = "PicsForGuide/5 IfYouAreinDCbottom.png" href = "Dev Container after Connecting" width = 300px />
+
+### Step 5
+After checking that you are in the Dev Container and locating the file you want to change, you can start looking for the Source Control icon, as seen below.
+
+<img src = "PicsForGuide/6 IfYouAreinDCOpenSC.png" href = "Source Control icon" width = 300px />
+
+### Step 6
+On top of ensuring you locate Source Control, you also need to make sure you are on the correct branch, or if you don't have a branch, how you could make one. To tell if you are on the correct branch, you can navigate back down to the bottom left of the VS code and look right next to the blue Dev container information. If it says main, you are in the wrong branch.
+
+<img src = "PicsForGuide/7 CheckBranch.png" href = "Main branch display"/>
+
+#### Below is the image guide with some text for making a new branch
+1. Click on where it displays the current branch name in the bottom left. Afterward, another drop-down menu will open, and you will click Create a new branch. If you want to change branches, you can select existing branches from the same drop-down menu.
+
+<img src= "PicsForGuide/8 MakeNewBranch.png" href = "Drop down for making new branches"/>
+
+2. Next, you would put in a branch name and hit enter, as shown below.
+
+<img src = "PicsForGuide/9 EnterBranchName.png" href= "Text box for inputting name"/>
+<img src = "PicsForGuide/10 BranchNameEntered.png" href = "Text box With branch name">
+3. Ensure you are on the correct branch by looking at the bottom left corner again, and it should look similar to the image below.
+
+<img src = "PicsForGuide/11 HowItshouldLookonBranch.png" href = "Dev Container with branch name"/>
+4. Finally, if you are on the Source Control Tab, it should look like the image below. (You can choose to publish the branch now or later; either is fine.)
+
+<img src = "PicsForGuide/12 howSCShouldLook.png" href = "Source Control with new branch" width = 300px />
+
+## Committing Your Changes
+
+### Step 7
+Start making changes whatever changes you need to make.
+### Step 8
+After making your changes, navigate to the Source Control tab, and it will probably look a bit like the image below.
+
+<img src = "PicsForGuide/13 afterChanges.png" href = "Source Control with all the changes made since last commt" width = 300px />
+
+Now, I know this looks like a lot of stuff. Probably a lot more stuff than what you remember changing. A lot of these files change because of how the Java extensions work. They constantly modify the file and change values for it to work, and it is imperative that you only choose the items you want to commit. But first, you have to locate them in this list.
```

Comment:
The next three paragraphs start with the word "Now". Vary your sentences and word usage.